sdk/storage/Azure.Storage.Common/src/Shared/AesGcm/AesGcmWindows.cs

```diff
@@ -0,0 +1,146 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+using System;
+using System.Runtime.InteropServices;
+using System.Security.Cryptography;
+
+namespace Azure.Storage
+{
+#if !(NETSTANDARD2_1_OR_GREATER || NETCOREAPP3_0_OR_GREATER)
```

Comment:
Did we plan to explicitly target netstandard2.1 as well? I don't see use adding that target to any of the libraries.

Reply:
My view is that you probably should. Otherwise you'll limit GCM for platforms implementing .NET Standard 2.1 to Windows only (which basically disables it for Xamarin) and would only work in Mono and Unity when running on Windows. Whether or not that's acceptable is up to you; adding a target definitely increases package size.

Reply:
@jaschrep-msft

Reply:
This statement is fine regardless of what our final targeting story is. Regardless of our targeting story, this prevents us from accidentally compiling this into any future splits, standard or official implementation.

Reply:
I agree that this condition is correct but right now it is a noop because we aren't targeting any of those higher platforms.